src/app/Scenes/MyCollection/Screens/Artwork/MyCollectionArtwork.tsx

```diff
@@ -86,20 +100,45 @@ const MyCollectionArtwork: React.FC<MyCollectionArtworkScreenProps> = ({
 
   const articles = extractNodes(artwork.artist?.articles)
 
-  const isEditable = !artwork.consignmentSubmission?.internalID
+  const isEditable =
+    !artwork.consignmentSubmission?.internalID ||
+    (artwork.consignmentSubmission?.internalID &&
+      ELIGIBLE_TO_EIDIT_STATES.includes(artwork.consignmentSubmission?.state))
```

Comment:
This could be simplified to: "it is editable if the artwork does not have an associated submission or the submission is in an editable state":

```suggestion
  const isEditable = !artwork.consignmentSubmission || ELIGIBLE_TO_EIDIT_STATES.includes(artwork.consignmentSubmission?.state))
```

Reply:
Agree with your comment @olerichter00. I do though think we should add this prop to Metaphysics and avoid maintaining it in Eigen